phylogenetic/build-configs/ci/copy_example_data.smk

```diff
@@ -1,10 +1,10 @@
 rule copy_example_data:
     input:
-        sequences="example_data/sequences.fasta",
-        metadata="example_data/metadata.tsv",
+        sequences="example_data/sequences_{segment}.fasta",
+        metadata="example_data/metadata_{segment}.tsv",
     output:
-        sequences="data/sequences.fasta",
-        metadata="data/metadata.tsv",
+        sequences="data/sequences_{segment}.fasta",
+        metadata="data/metadata_{segment}.tsv",
```

Comment:
Good point, I'll go ahead and refactor the example data. 

Just a note that lassa seems a bit different than pathogens like dengue in that we might have one strain with multiple GenBanks by segment (e.g. the Josiah strain has two NCBI reference sequence: HQ688672 (S-segment); JN650518 (L-segment)). Ergo, I'll just combine both L and S Genbanks from [here (L)](https://github.com/nextstrain/lassa/blob/master/example_data/metadata_l.tsv) and [here (S)](https://github.com/nextstrain/lassa/blob/master/example_data/metadata_s.tsv) into one example dataset.

Reply:
Ah, thanks for the note. I guess follow up question: should the separation of segments be done in the ingest workflow? (similar to how dengue separates into the 4 serotypes)

Reply:
> should the separation of segments be done in the ingest workflow?

I was grappling with this question 😄 

The rules are in a separate "prepare_sequence_segments.smk" so it's easily moveable to the ingest section if needed. To be similar to the dengue workflow, I was trying to figure out how to get L and S coverage when they are composed of multi-gene segments (see [example](https://github.com/nextstrain/lassa/blob/c04cb161b503e1a7af5f5f40658a1a6e2337cde9/config/lassa_l.gb#L47-L59)), perhaps a specialized script... but I figured that work might be better scoped in a dedicated separate PR. Open to suggestions here though!

Reply:
Ah, so lassa would be more analogous to influenza where the GenBank records are already labeled/separated by segments.

Then you might want to consider following the avian-flu ingest example of pulling data directly from NCBI Virus so that the [segment field is available](https://github.com/nextstrain/avian-flu/blob/c6a8fcf0bbd1e84e7acf93f7b06d13e07d129510/ingest/build-configs/ncbi/bin/ncbi-virus-url#L66).

Reply:
Segment separation moved to the ingest workflow, rebased commits accordingly.